frontend/src/components/maps/leaflet/MapProperties.test.tsx

```diff
@@ -44,13 +46,12 @@ const mockDetails: IParcelDetail = {
     isSensitive: false,
     evaluations: [
       {
-        assessedValue: 1000000,
-        estimatedValue: 0,
-        fiscalYear: 2019,
-        netBookValue: 0,
-        appraisedValue: 0,
+        date: '2019',
+        key: '',
```

Comment:
Did you want to test an empty string?  I thought I created enums that you could use.

Reply:
This is just to make this test compile, I'll write actual tests for this change when I start working on the parcelDetailForm updates.